### PR TITLE
Generate RGB histogram from an image

### PR DESCRIPTION
**Pull request by RivkaZishinski**

### CODE REVIEW

**Comment by bdsoha** [comment]
@RivkaZishinski Where is the *PR* description?

**Comment by bdsoha** [comment]
🎉 Great work on being the 2nd PR.

### Few Cleanup Tasks
- [ ] There are *merge conflicts* because you did not create a `feature/***` branch from `main`.
- [ ] Do not overwrite content on the `main` branch.
- [ ] Name your repository that same as the *forked* repository.
- [ ] Move your code into a sub-directory.

### Additions
- [ ] Add a `README.md` describing how to use/install your code.

**Comment by RivkaZishinski** [comment]
I use with path specific and want to know-
if i need to to it for all images where to put the images in docker? in VS

**Comment by bdsoha** [comment]
This assignment is an exercise.
Feel free to map a hard-code path to an image into the docker container or write a small `input` to capture a user-defined path.

File: RGBByNumpy.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+from PIL import Image
+
+img = Image.open('C:/Users/RivkaZ/Documents/GitHub/numpy/light_blue.png')
```
Comment (bdsoha) [inline RGBByNumpy.py:6]: I noticed that you reverted changes to a hard-coded path, *(which is 👌)*.
You just need to be sure that from within the running docker container you have access to that path *(using `C:` won't work)*.

**Comment by RivkaZishinski** [comment]
I real  got stuck on the eror of the fath full time .
using C: won't work
i bring the image to the directy of all project but it is not good

**Comment by bdsoha** [comment]
Are you using `docker run ...`?

**Comment by RivkaZishinski** [comment]
yes.
docker run rgb-by-numpy

**Comment by bdsoha** [comment]
From within the directory containing the image you want to use, run the following:
```bash
docker run -it --rm -v %cd%:/assets rgb-by-numpy
```

Then you can reference the image path at `/assets/light_blue.png`

**Comment by RivkaZishinski** [comment]
it bring error i will try more

**Comment by bdsoha** [comment]
> it bring error i will try more

Are you running `cmd` or `powershell`?

**Comment by RivkaZishinski** [comment]
powershell -
can I send a powershell

**Comment by bdsoha** [comment]
For powershell:

```powershell
docker run -it --rm -v ${PWD}:/assets rgb-by-numpy
```

**Comment by RivkaZishinski** [comment]
thank. 
שמירת התרגום
I'll deal with that later, now I want to move to-
Few Cleanup Tasks
I want to make sure I understand what to do.
Fork: skyvar/registration ➡RivkaTzishinsky/registration.
Branch: RivkaTzishinsky/registration:main ➡ RivkaTzishinsky/registration:feature/numpy.

**Comment by bdsoha** [comment]
- What is your current fork name? It should be `registration`.
- What is your fork's main branch name? It should be `main`.
- What is your feature fork name? It should be `feature/rgb-histogram`.

**Comment by bdsoha** [comment]
@RivkaZishinski I see that your new fork is misspelled and does not contain a `main` branch.

**Pull request closed without merging**